db/migrate/20160208173646_add_the_visit_id_to_models.rb

```diff
@@ -0,0 +1,19 @@
+class AddTheVisitIdToModels < ActiveRecord::Migration
+  def up
+    add_column :proposals, :visit_id, :uuid
+    add_column :comments, :visit_id, :uuid
+    add_column :reports, :visit_id, :uuid
+    add_foreign_key :proposals, :visits
+    add_foreign_key :comments, :visits
+    add_foreign_key :reports, :visits
+  end
+
+  def down
+    remove_foreign_key :proposals, :visits
+    remove_foreign_key :comments, :visits
+    remove_foreign_key :reports, :visits
+    drop_column :proposals, :visit_id
```

Comment:
Agreed. I only noticed this after trying to `rake db:rollback` on c2-dev after QA was complete.